node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -319,6 +320,9 @@ impl Initialized {
 					},
 				);
 			}
+
+			// Decrement spam slots for freshly backed or included candidates
+			self.reduce_spam_on_backed_included(candidate_events);
```

Comment:
If I remember correctly - we don't need this anymore. Spam slots are cleared in `process_on_chain_votes` called a few lines above. Just add a comment explaining this side effect.

Reply:
Hmm. I think `process_on_chain_votes` only clears spam slots in the backed case. Still, if we're sure the backed case is handled sufficiently there then I can remove it here. Also that would let me refactor the filtering of events to happen in `scraping/mod.rs`, which would be much nicer.

Reply:
You are right that only backing votes are processed in `process_on_chain_votes`.
But generally I'm wrong here. Handling backing votes is different from handling backing events. We handle the former (as you said) but we won't handle the latter if we remove this line.

Bottom line - it should stay.

Reply:
Addressed this with an implementers guide explanation and removed the associated code.